backend/arm64/emit.mlp

```diff
@@ -863,6 +864,10 @@ let emit_instr i =
                      | Ispecific Isqrtf -> "fsqrt"
                      | _                -> assert false) in
         `	{emit_string instr}	{emit_reg i.res.(0)}, {emit_reg i.arg.(0)}\n`
+    | Lop(Ivalueofint | Iintofvalue) ->
+        let src = i.arg.(0) and dst = i.res.(0) in
+        if src.loc <> dst.loc then
+        `	mov	{emit_reg dst}, {emit_reg src}\n`
```

Comment:
I don't think we know for sure it
will be a move; one operand could
be on the stack.

Reply:
(Basically, I think it should follow
https://github.com/ocaml-flambda/flambda-backend/blob/main/backend/arm64/emit.mlp#L674)